mne_bids/write.py

```diff
@@ -16,14 +16,15 @@
 import numpy as np
 from numpy.testing import assert_array_equal
 import mne
-from mne import Epochs
+from mne import Epochs, pick_types, find_events, events_from_annotations
 from mne.io.constants import FIFF
 from mne.io.pick import channel_type
 from mne.io import BaseRaw
 from mne.channels.channels import _unit2human
 from mne.utils import check_version, has_nibabel
 from mne.transforms import _ensure_trans, apply_trans
 
+from pybv import write_brainvision
```

Comment:
I would not do a hard dependency. Only if you are attempting to write EEG data would I import this